packages/devtools/lib/src/tables.dart

```diff
@@ -834,6 +873,31 @@ abstract class TreeColumn<T extends TreeNode<T>> extends Column<T> {
   }
 }
 
+class TreeTableToolbar<T extends TreeNode<T>> extends CoreElement {
+  TreeTableToolbar() : super('div') {
+    add(div(c: 'btn-group')
+      ..add([
+        PButton('Expand all')
+          ..small()
+          ..click(_expandAll),
+        PButton('Collapse all')
+          ..small()
+          ..click(_collapseAll),
+      ]));
+  }
+
+  TreeTable<T> _treeTable;
+  set treeTable(TreeTable table) => _treeTable = table;
```

Comment:
How about making this a public field instead?

https://dart.dev/guides/language/effective-dart/design#dont-define-a-setter-without-a-corresponding-getter